src/views/health-checks.vue

```diff
@@ -3,9 +3,9 @@
         <div v-if="compatible">
             <h1>Health checks</h1>
```

Comment:
In Firefox, this popup is not scrollable after displaying the list of health checks in BlendingEditor:

![image](https://user-images.githubusercontent.com/566463/36484647-38c39fb2-171a-11e8-803e-3fe87997ff18.png)